assets/html/volunteer.html

```diff
@@ -0,0 +1,373 @@
+            (technical
+            and not)
+          </span> and energy. Use language such as <span>"yes/and,"</span> not "no/but."</li>
+        <li><span>Encourage</span> members and participants to <span>listen</span> as much as they speak.</li>
+        <li>Strive to build tools that are <span>
+            open and free technology for public
+            use</span>.</li>
+        <li><span>Prioritize access</span> for and input from those who are <span>
+            traditionally
+            excluded from the civic process</span>.</li>
+        <li>Work to ensure that the community is <span>well-represented</span> in the
+          planning, design, and implementation of civic tech. This includes
+          <span>encouraging participation</span> from <span>
+            women, minorities, and traditionally
+            marginalized groups
+          </span>.</li>
+        <li><span>Actively involve community groups</span> and those with subject matter
+          expertise in the decision-making process.</li>
+        <li>Ensure that the <span>relationships</span> and conversations between <span>
+            community
+            members
+          </span>, the <span>local government</span> staff and <span>community partners</span> remain
+          <span>respectful, participatory, and productive</span>.</li>
+        <li>Provide an environment where people are <span>
+            free from discrimination
+            or harassment
+          </span>.</li>
+      </ol>
+      <p>
+        Code for San José reserves the right to ask anyone in violation of
+        these policies not to participate in Code for America network
+        activities, events, and digital forums.
+      </p>
+
+      <h3>Anti-Harassment Policy</h3>
+      <p>
+        This anti-harassment policy is based on the example <a
+          href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy" target="_blank"
+          rel="noopener noref">policy from the
+          Geek Feminism wiki</a>, created by the Ada Initiative and other
+        volunteers.
+      </p>
+      <p>
+        This policy is based on several other policies, including the Ohio
+        LinuxFest anti-harassment policy, written by Esther Filderman and Beth
+        Lynn Eicher, and the Con Anti-Harassment Project. Mary Gardiner,
+        Valerie Aurora, Sarah Smith, and Donna Benjamin generalized the
+        policies and added supporting material. Many members of LinuxChix,
+        Geek Feminism and other groups contributed to this work.
+      </p>
+      <p>
+        All Code for San José network activities, events, and digital forums
+        and their staff, presenters, and participants are held to an
+        anti-harassment policy, included below.
+      </p>
+      <p>
+        In addition to governing our own events by this policy, Code for San
+        Joséwill only lend our brand and fund groups that offer an
```

Comment:
Add a space between the words  "José" and "will"

Reply:
Great catch! My first attempts at find and replace had mixed results. :P